### ISSUE TITLE
Can't get data from websocket

### ISSUE THREAD

**Issue by yusufozcan** [dutu/poloniex-api-node#48]
It is logging that "Poloniex websocket connected" but there is no incoming messages altough subscribed.

EDIT:
I checked examples and version 2 is working. sorry.

**Comment by dutu** [comment]
@yusufozcan , I have done a couple of tests a few days ago and I haven't received data from Poloniex over WAMP push API (v1), while websocket v2 API works OK.

Lack of data when using WAMP (v1) seems to be an issue on the Poloniex side, since the very same tests I have done a couple of weeks ago were successful.

**Comment by dutu** [comment]
I have reported the issue with no data sent over WAMP push API to Poloniex support on 31 Jan.
Still awaiting their response...

**Comment by MoroZvlg** [comment]
Any changes?

**Comment by dutu** [comment]
No news, sorry

**Comment by aryeharmon** [comment]
any news yet?

**Comment by dutu** [comment]
There is no update. You can use WebSocket v2 API instead.

**Comment by qutozze** [comment]
I use python websockets lib for getting data from websockets v2, but I check your js code in terms of ws connecting and subscribing and I found and there are identical. 
But I've faced the problem, after connect to ws I start receiving messages: [1010] and that is all.
I tried to connect to channels: 1001, 1002, 1003, "BTC_XMR" - anyway.
Have you face such problems? Some tips would be very helpful for me.. Thanks
